lib/facter/resolvers/mountpoints.rb

```diff
@@ -67,23 +67,31 @@ def get_mount_data(file_system, mount)
         end
 
         def get_mount_sizes(mount)
-          stats = Facter::Util::Resolvers::FilesystemHelper.read_mountpoint_stats(mount[:path])
+          begin
+            stats = Facter::Util::Resolvers::FilesystemHelper.read_mountpoint_stats(mount[:path])
+            get_bytes_data(mount, stats)
+          rescue Sys::Filesystem::Error => e
+            @log.debug("Could not get stats for mountpoint #{mount[:path]}, got #{e}")
+            mount[:size_bytes] = mount[:available_bytes] = mount[:used_bytes] = 0
```

Comment:
Does it help if we add this facts with value `0`? since this is not the real information.
What happens if we do not set a value for this fields?

Reply:
I think it is for consistency, we have similar behavior in osx resolver: https://github.com/puppetlabs/facter/blob/64502d3561da0db674ccfdb9f0d9a3f3d70cde07/lib/facter/resolvers/macosx/mountpoints.rb#L43

Reply:
It will fail when trying to calculate total_bytes, https://github.com/puppetlabs/facter/blob/main/lib/facter/resolvers/mountpoints.rb#L74 and the fact will return nil.
I added the 0 values as the mountpoint fact already had some real data in it, like mount[:device], mount[:filesystem], mount[:path] and mount[:options]